infrastructure/templates/public-api/components/alerts/baseCpuPercentageAlert.bicep

```diff
@@ -0,0 +1,38 @@
+import { Severity } from 'types.bicep'
+
+@description('Names of the resources that these alerts are being applied to.')
+param resourceNames string[]
+
+@description('Names of the resources that these alerts are being applied to.')
+param resourceType string
+
+@description('Names of the resources that these alerts are being applied to.')
+param metricName string
```

Comment:
As above - could create a `CpuMetricName` type that could assist in auto-completion and general type safety.